presto-jmx/src/main/java/io/prestosql/plugin/jmx/JmxRecordSetProvider.java

```diff
@@ -153,7 +153,12 @@ else if (value.getClass().getComponentType() == short.class) {
     }
 
     @Override
-    public RecordSet getRecordSet(ConnectorTransactionHandle transaction, ConnectorSession session, ConnectorSplit split, ConnectorTableHandle table, List<? extends ColumnHandle> columns)
+    public RecordSet getRecordSet(
+            ConnectorTransactionHandle transaction,
+            ConnectorSession session,
+            ConnectorSplit split,
+            ConnectorTableHandle table,
+            List<? extends ColumnHandle> columns)
```

Comment:
Unnecessary change for jmx migration. Basically, we separate the commit to fix styles.

Reply:
I can't remember. Let me revert this change.